tests/vmi_multus_test.go

```diff
@@ -698,6 +698,25 @@ var _ = Describe("[Serial]SRIOV", func() {
 			}
 		}
 
+		It("[Serial]should create virt-launcher pod with CAP_SYS_RESOURCE capability", func() {
```

Comment:
- The first container is marked as `[serial]` already. You do not need to mark it again here.
- Why do we need a special test for this? It will be cheaper to just add another assert to an existing one.
For example to "[test_id:1754]should create a virtual machine with sriov interface".

You can call from it to an assert helper, e.g. `assertVirtLauncherIncludesCapability(vmi, ...)

Reply:
1. This test make use of `KubevirtDefaultConfig` ([comment](https://github.com/kubevirt/kubevirt/pull/4874/files/03d29a67d49a89760b6f7eb574ab3f19f0cbc9fe..f5de62693873e4ce1b483e112e71ba4a6673e1c7#r569567253)), it is expected a test to ran as serial.
I added `[serial]` for this test explicitly to make sure this test will run as serial in case it will be moved
or changed.

2. I dont think we should piggyback this test on another one, in my opinion its too cheap.
Also one could ask why do we assertVirtLauncherIncludesCapability only on the first test.
This test works and didnt took such effort anyway.
What are your concerns with having this test?

Reply:
1. There are other reasons why these tests cannot run in parallel and this specific one exists in other places without such an explicit mark.
2. An e2e test is not cheap, it can take a few minutes to run the test.